Cargo.toml

```diff
@@ -7,7 +7,6 @@ members = [
     "compiler/noirc_driver",
     "compiler/noirc_printable_type",
     "compiler/fm",
-    "compiler/wasm",
```

Comment:
We have the source-resolver in compiler/wasm -- not really convinced that source-resolver is a compiler artifact. You could make the argument that its kind of the wasm/ts code for compiler/fm